util/package.go

```diff
@@ -316,9 +315,32 @@ func (p *Package) Validate() error {
 		}
 	}
 
+	err = p.validateVersionConsistency()
+	if err != nil {
+		return errors.Wrap(err, "version in manifest file is not consistent with path")
+	}
+
 	return p.ValidateDatasets()
 }
 
+func (p *Package) validateVersionConsistency() error {
+	versionPackage, err := semver.Parse(p.Version)
+	if err != nil {
+		return errors.Wrap(err, "invalid version defined in manifest")
+	}
+
+	baseDir := filepath.Base(p.BasePath)
+	versionDir, err := semver.Parse(baseDir)
+	if err != nil {
+		return nil // package content is not rooted in version directory
```

Comment:
Do you expect us to support this long term or only at the moment?

Reply:
I planned to leave it there long term. I put this check just in case.